project/source/game/Ai.cpp

```diff
@@ -3012,7 +3013,7 @@ void Game::AI_Shout(LevelContext& ctx, AIController& ai)
 }
 
 //=================================================================================================
-// je�li target jest nullptr to atak nic nie zadaje (trenuje walk� na manekinie)
+// attack does not imply damage for nullptr target (dummy training)
```

Comment:
when target is nullptr it deals no damage (dummy training)

Reply:
I think my version sounds better better but ok